src/components/navbar/index.tsx

```diff
@@ -94,6 +96,19 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
       >
         Settings
       </Menu.Item>
+      <Menu.Item
+        onClick={() => {
+          if (asyncRequestIsComplete(tokens)) {
+            AuthClient.logout(tokens.result.refreshToken).then(() => {
+              localStorage.removeItem(LOCALSTORAGE_STATE_KEY);
+            });
+            history.push(Routes.HOME);
+            history.go(0); // force refresh if user is already on home page
```

Comment:
force refresh here is actually pretty big brain, I kind of like it 😄 

I think we should still use the thunk from master but this is super creative